src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,51 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private static final MovieService MOVIE_SERVICE =
+            (MovieService) INJECTOR.getInstance(MovieService.class);
+    private static final CinemaHallService CINEMA_HALL_SERVICE =
+            (CinemaHallService) INJECTOR.getInstance(CinemaHallService.class);
+    private static final MovieSessionService MOVIE_SESSION_SERVICE =
+            (MovieSessionService) INJECTOR.getInstance(MovieSessionService.class);
```

Comment:
There is no need to make them CONSTANTS, you can create them where needed inside the `main` method.